Support CoffeeScript 2+

ylxdzsw opened the pull request:
So this is the @peko 's fix in #14 .

ForbesLindesay commented on package.json:
```diff
@@ -9,7 +9,7 @@
   ],
   "dependencies": {
-    "coffee-script": "~1.12.7",
+    "coffeescript": ">=2",
```
Please use the `^2.4.1` style to refer to version 2. It shouldn't automatically pull in version 3 if a coffee-script version 3 is ever released.

ForbesLindesay commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "jstransformer-coffee-script",
-  "version": "1.1.1",
+  "name": "jstransformer-coffee",
```
Can we stick to the old name. That way when people upgrade their dependencies, they will automatically get upgraded, rather than having to manually seek out the new package.

ylxdzsw commented:
Hi, since I don't have control over @peko 's repo, I have forked and opened another pull request #16 where I addressed your comments, please take a look.
